hudi-common/pom.xml

```diff
@@ -261,10 +265,6 @@
           <groupId>tomcat</groupId>
           <artifactId>*</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>org.apache.hbase.thirdparty</groupId>
-          <artifactId>hbase-shaded-jetty</artifactId>
-        </exclusion>
```

Comment:
cc @yihua for the review~

Reply:
Hi @danny0405, thanks for helping out but this PR is still WIP.

Hudi uses `hbase-shaded-netty` from hbase thirdparty package and the netty is outdated there. Even the latest OSS hbase thirdparty won't have the fixed netty version. I'm still trying to figure out how to exclude netty from that shaded package